cobra.go

```diff
@@ -40,14 +40,6 @@ var initializers []func()
 // Set this to true to enable it
 var EnablePrefixMatching bool = false
 
-// enables an information splash screen on Windows if the CLI is started from explorer.exe.
-var EnableWindowsMouseTrap bool = true
```

Comment:
1. preExecHook is a private function, so I can't overwrite it from outside
2. This is an API change vs the variable. So while I think this is a better API, I'd rather not redo it unless we need to.